server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -5109,4 +5108,15 @@ public IpQuarantineResponse createQuarantinedIpsResponse(PublicIpQuarantine quar
 
         return quarantinedIpsResponse;
     }
+
+    private String getDomainPath(String path){
+        StringBuilder domainPath = new StringBuilder();
+        if(ObjectUtils.isEmpty(path) || path.equals("/")){
```

Comment:
IMO. Since we know that path is a string, better to use `org.apache.commons.lang3.StringUtils.isEmpty()`..